.github/workflows/main.yml

```diff
@@ -45,7 +49,7 @@ jobs:
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
-          username: ${{ github.repository_owner }}
+          username: ${{ github.actor }}
```

Comment:
According to [docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs), actor is "The username of the user that triggered the initial workflow run."

Doesn't this vary too often?
Shouldn't we use the repository_owner instead?

Reply:
Although, I saw that it's suggested in the docs, so let's try it